code/game/objects/structures/beds_chairs/bed.dm

```diff
@@ -88,15 +88,16 @@
 /obj/structure/bed/roller/attack_hand_secondary(mob/user, list/modifiers)
 	. = ..()
 	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
-		return
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 	if(!ishuman(user) || !user.canUseTopic(src, BE_CLOSE))
-		return FALSE
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 	if(has_buckled_mobs())
-		return FALSE
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
```

Comment:
> Also add `return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN` or `return SECONDARY_ATTACK_CONTINUE_ATTACK_CHAIN` at the end.
> 
> I really need to change all these to not be `. = ..()` and instead check `if (..() == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)` directly, and grep for it.

looks like mothblocks suggested it

Reply:
> Why do we cancel here instead of continue?

I honestly just wanted to make sure this PR works without headaches so I went with cancel, I can swap for continue if it would be preferable.

Reply:
that's fine, just wanted to know your reasoning.